data/json/furniture_and_terrain/furniture-roof.json

```diff
@@ -82,9 +86,7 @@
     "move_cost_mod": 2,
     "required_str": 8,
     "flags": [ "TRANSPARENT", "BASHABLE" ],
-    "deconstruct": {
-      "items": [ { "item": "pipe", "count": 6 }, { "item": "steel_chunk", "count": 2 }, { "item": "cable", "charges": 10 } ]
-    },
+    "deconstruct": { "items": [ { "item": "pipe", "count": 6 }, { "item": "steel_chunk", "count": 2 }, { "item": "pipe_fittings", "count": [ 1, 4 ] }, { "item": "cable", "charges": 10 } ] },
```

Comment:
"deconstruct": {
      "items": [
        { "item": "pipe", "count": 6 },
        { "item": "steel_chunk", "count": 2 },
        { "item": "pipe_fittings", "count": [ 1, 4 ] },
        { "item": "cable", "charges": 10 }
      ]
    },

Reply:
Check with the web linter, http://dev.narc.ro/cataclysm/format.html

Reply:
Ran through it again - I've already checked with the web linter(second commit, 4th and 5th). Not quite sure what's going wrong. Going to repost file directly into github web from the linter.

Reply:
This time it worked. Guess I'm going to have to use the web interface from here on out. Thank you!